conjur/controller/login_controller.py

```diff
@@ -93,13 +92,7 @@ def get_api_key(self, conjurrc):
         """
         Method to fetch the user/host's API key from Conjur which is to be added to the netrc
         """
-        try:
-            self.credential_data.api_key = self.login_logic.get_api_key(self.ssl_verify,
-                                                                        self.credential_data,
-                                                                        self.user_password,
-                                                                        conjurrc)
-        except CertificateVerificationException:
-            raise
```

Comment:
why removing this?

Reply:
because it is caught in the `except Exception as server_error` of the `cli.py` so we don't need this

Reply:
Now see this is just catch and throw the same so really no needed